hello_world/transactions/test/hello_transaction.test.js

```diff
@@ -0,0 +1,91 @@
+const HelloTransaction = require('../hello_transaction');
+const {TransactionError, utils} = require('@liskhq/lisk-transactions');
+const { when } = require('jest-when');
+
+describe('Hello Transaction', () => {
+    let storeStub;
+    beforeEach(() => {
+        storeStub = {
+            account: {
+                get: jest.fn(),
+                set: jest.fn(),
+            },
+        };
+    });
+
+    test('it should save the hello string in the senders account assets', async () => {
+        // Arrange
+        const asset = {
+            hello: "my hello message",
+        };
+
+        const senderId = '16313739661670634666L';
+        const senderPublicKey = 'c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f';
+        const mockedSenderAccount = {
+            address: '16313739661670634666L',
+            publicKey: 'c094ebee7ec0c50ebee32918655e089f6e1a604b83bcaa760293c61e0f18ab6f',
+            asset: {},
+        };
+
+        when(storeStub.account.get)
+            .calledWith(senderId)
+            .mockReturnValue(mockedSenderAccount);
+
+        // Act
+        const tx = new HelloTransaction({
+            senderPublicKey,
+            asset,
+            timestamp: utils.getTimeFromBlockchainEpoch(new Date()),
+        });
+        tx.applyAsset(storeStub);
```

Comment:
maybe it's good practice to call `tx.apply` instead, so that the whole execution is tested

Reply:
we'll keep `applyAsset()` for now, as discussed :)